lib/node/Creator.js

```diff
@@ -4,45 +4,64 @@
   information on nodes in mathJS
 */
 
-const math = require('mathjs');
 const NodeType = require('./Type');
 
+const opFns = {
+  '+': 'add',
+  '-': 'subtract',
+  '*': 'multiply',
+  '/': 'divide',
+  '^': 'pow',
+};
+
+const operatorNode = (op, fn, args) => {
```

Comment:
I was assuming this would live in `math-parser` instead of `mathsteps` (because `mathjs` did both as well)

what are your thoughts on where it should go and why?